src/android/CameraLauncher.java

```diff
@@ -297,27 +311,30 @@ public void takePicture(int returnType, int encodingType)
                 applicationId + ".cordova.plugin.camera.provider",
                 photo);
         intent.putExtra(MediaStore.EXTRA_OUTPUT, imageUri);
-        //We can write to this URI, this will hopefully allow us to write files to get to the next step
+        // We can write to this URI, this will hopefully allow us to write files to get
+        // to the next step
         intent.addFlags(Intent.FLAG_GRANT_WRITE_URI_PERMISSION);
 
         if (this.cordova != null) {
-            // Let's check to make sure the camera is actually installed. (Legacy Nexus 7 code)
+            // Let's check to make sure the camera is actually installed. (Legacy Nexus 7
+            // code)
             PackageManager mPm = this.cordova.getActivity().getPackageManager();
-            if(intent.resolveActivity(mPm) != null)
-            {
+            if (intent.resolveActivity(mPm) != null) {
                 this.cordova.startActivityForResult((CordovaPlugin) this, intent, (CAMERA + 1) * 16 + returnType + 1);
-            }
-            else
-            {
+            } else {
                 LOG.d(LOG_TAG, "Error: You don't have a default camera.  Your device may not be CTS complaint.");
+                this.failPicture("No Camera available.");
```

Comment:
@robin-hartmann should we localize this message in app?

Reply:
> @robin-hartmann should we localize this message in app?

Absolutely. I think for this purpose we should rename `DevicesNotFoundError` to `CameraNotAvailable` in `packages/pal/msb-native-api/src/Camera/Contract/CameraError.ts` with the message `Camera not available` and then use that error. Also, we should rename the respective translation key `cameraDevicesNotFound` to `cameraNotAvailable`. The translation text already seem to fit.